controllers/front/CartController.php

```diff
@@ -662,7 +677,7 @@ private function checkCartProductsMinimalQuantities()
                 $this->errors[] = $this->trans(
                     'The minimum purchase order quantity for the product %product% is %quantity%.',
                     [
-                        '%product%' => $product['name'],
+                        '%product%' => $product['name'] . (isset($product['attributes']) ? ' ' . $product['attributes'] : ''),
```

Comment:
I think $product['attributes'] is an array here

Reply:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Cart.php#L830
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Product.php#L5708